azure-functions/acbs-function/mappings/facility/facility-loan.js

```diff
@@ -40,10 +40,10 @@ const facilityLoan = (deal, facility, acbsData) => {
     productTypeId: helpers.getProductTypeId(facility, deal.dealSnapshot.dealType),
     productTypeGroup: helpers.getProductTypeGroup(facility, deal.dealSnapshot.dealType),
     currency: facility.facilitySnapshot.currency.id,
-    amount: facility.tfm.ukefExposure,
+    amount: facility.tfm.ukefExposure || facility.facilitySnapshot.ukefExposure,
```

Comment:
I'm not sure if `facility.tfm.ukefExposure` even exists?

Reply:
Yep it does (GEF)

![image](https://user-images.githubusercontent.com/32125108/157547449-409e06c0-434c-477a-a654-529d264c3520.png)